draft-dss-star.md

```diff
@@ -868,14 +858,12 @@ the Aggregation Server. There are several types of bogus reports:
 - Reports with invalid shares, or corrupt reports. These are reports that will yield the incorrect
   secret when combined by the Aggregation Server.
 - Reports with invalid ciphertext, or garbage reports. These are reports that contain an encryption
-  of the wrong measurement value (`msg`).
+  of the wrong measurement value (`secret`).
 
-Corrupt reports can be mitigated by using a verifiable secret sharing scheme, such as the one described
-in {{dep-vss}}, and verifying that the share commitments are correct for each share. This ensures that
-each share in a report set corresponds to the same secret.
+Given that the Aggregation Server creates a report set by grouping reports together by commitment, it is possible for a client to mount a Denial-of-Service (DoS) attack against a server in the Aggregation phase ({{aggregation-phase}}) by submitting an incorrect share for a particular commitment, which will cause the recovery process for that report set to fail. To prevent this attack, a verifiable secret sharing scheme, such as the one described in {{dep-vss}}, can be used to verify that the share commitments are correct for each share. This ensures that each share in a report set corresponds to the same secret.
```

Comment:
This is a good clarification, but it comes across as somewhat unspecified behavior. As an implementer, do I need to use a VSS scheme? And if so, how do I negotiate it? (I think these are configured things.) I have an idea for how to address this that I can send in a followup PR.